dicomanonymizer/anonymizer.py

```diff
@@ -7,6 +7,16 @@
 
 from .simpledicomanonymizer import *
 
+
+def isDICOMType(filePath):
+    """
+    :returns True if input file is a DICOM File. False otherwise.
+    """
+    with open(filePath, 'rb') as tempFile:
+        tempFile.seek(0x80, os.SEEK_SET)
```

Comment:
what if file is smaller than 0x80 ?

Reply:
It moves where we want and when we read, it will be an empty string.